pipelines/_ingredients/ApPipe.yaml

```diff
@@ -101,6 +101,37 @@ subsets:
       - initialPviCore
     description: >
       An alias of ApPipe to use in higher-level pipelines.
+  preload:
+    subset:
+      - loadDiaCatalogs
+    description: Tasks that can be run before receiving raw images.
+  prompt:
+    subset:
+      - isr
+      - calibrateImage
+      - retrieveTemplate
+      - subtractImages
+      - detectAndMeasure
+      - filterDiaSrcCat
+      - rbClassify
+      - transformDiaSrcCat
+      - diaPipe
+    description: >
```

Comment:
Please add a few of the metrics tasks to the `prompt` subset, since they are useful to have in the Prompt Processing Chronograf dashboard:
` - initialPviCore`
` - diffimTaskCore`
` - analyzeAssocDiaSrcCore`
` - analyzeTrailedDiaSrcCore`

Reply:
To confirm, `injectedMatch` still belongs in the afterburner?

Reply:
No, it had better go in the `prompt` subset. It won't usually matter, but if it does we will want the Fakes metrics before the afterburner runs.